lib/commons/matches/from-definition.js

```diff
@@ -0,0 +1,48 @@
+/* global matches */
+const matchers = [
+  'nodeName',
+  'attributes',
+  'properties',
+  'condition'
+]
+
+/**
+ * Check if a node matches some definition
+ * 
+ * Example:
+ * ```js
+ * matches.fromDefinition(node, {
+ *   nodeName: ['div', 'span']
+ *   attributes: {
+ *     'aria-live': 'assertive'
+ *   }
+ * })
+ * ```
+ * 
+ * @param {HTMLElement|VirtualNode} node
+ * @param {Object|Array<Object>} definition
+ * @returns {Boolean}
+ */
+matches.fromDefinition = function matchFromDefinition (node, definition) {
+  node = node.actualNode || node;
+	if (Array.isArray(definition)) {
+		return definition.some(definition => matches(node, definition));
```

Comment:
The reuse of the word `definition` as a variable in `.some` makes the recursive call a bit difficult to follow. Can you refine it to describe more accurately what is being provided as an argument in the recursive call to `matches`?